DEVGUIDE.md

```diff
@@ -152,13 +152,13 @@ components installed into Visual Studio 15.  You can revert this step by disabli
 
 For **Debug**:
 
-1. Ensure that the VSIX package is uninstalled. In VS, select Tools/Extensions and Updates and if the package `VisualStudio.FSharp.EnableOpenSource` is installed, select Uninstall
-1. Run ``debug\net40\bin\EnableOpenSource.vsix``
+1. Ensure that the VSIX package is uninstalled. In VS, select Tools/Extensions and Updates and if the package `Visual F# Tools` is installed, select Uninstall
+1. Run ``debug\net40\bin\VisualFSharpVsix.vsix``
```

Comment:
DEVGUIDE was already updated to note the VSIX changes. Devs won't use the MSI during development as its only purpose is to be inserted into VS.